### ISSUE TITLE
CT scrolling is lagging since Big sur and horos update

### ISSUE THREAD

**Issue by nicodu74** [horosproject/horos#623]

**Comment by techmdmac** [comment]
Yes, very annoying. The same with the issue "Clipping 3D Volume Rendering #620"

**Comment by nikaonashvili** [comment]
ct scrolling is lagging in Big Sur and latest Horos. it happens only in 2d mode, it resolves in mar. in volume rendering for me its ok.

**Comment by techmdmac** [comment]
3D clipping works normally to you ? See issue 620. Also, when you save a 3D VR image as DICOM serie , that image is saved displaced of the center of the screen. Can you check it? Thank you.
